script/reset-environment.sh

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/env bash
 if ! [ -x "$(command -v yarn)" ]; then
     echo "Installing yarn..."
-    npm i -g yarn@0.21.3
+    npm i -g yarn@0.27.5
     if [ $? -eq 0 ]; then
         echo "Yarn successfulling installed globally."
     else 
         echo "Yarn failed to install...please install manually."
         exit 1
     fi
+else
+    yarn_version=$(yarn --version)
+    if [ "$yarn_version" != "0.27.5" ]; then
+        echo "Install yarn version 0.27.5 [y/N]: "
```

Comment:
Just for fun, can we also display the current version of yarn (to make it obvious why we're asking)?

```bash
echo "Current yarn version is $yarn_version. Install yarn version 0.27.5 [y/N]: "
```
Or something.